class-tgm-plugin-activation.php

```diff
@@ -2093,6 +2095,19 @@ public static function get_instance() {
 
 			return self::$instance;
 		}
+
+		/**
+		 * Adds css to admin head
+		 *
+		 * @since 2.6.2
+		 */
+		function tgm_css() {
```

Comment:
I'd like this to be renamed to `admin_css` (and the hook reference), just to be a little more specific.